starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +10,98 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+/* ---- reset ---- */
+
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+/* --- typography ---- */
+
+#title h1{
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 72px;
+  text-align: center;
+  color: white;
+  padding-top: 20%;
+  font-weight: lighter;
+}
+div p{
+  text-align: left;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 22px;
+  width: 70%;
+  margin-left: 15%;
+  margin-top: 10%;
+  color: grey;
+  font-weight: lighter;
+}
+
+/* --- layout --- */
+
+.main-container{
+  margin-left: 20%;
+  width: 60%;
+  margin-top: 5%;
+}
+#title{
+  background-image: url('../images/soup.jpg');
+  height: 600px;
+  width: 100%;
+  background-size: cover;
+}
+#recipeinfo{
+  margin-left: 35%;
+  width: 30%;
+}
+#cookinginfo{
+  margin-left: 25%;
+  width: 50%;
+}
+div hr{
+  width: 70%;
+  margin-top: 5%;
+}
+.ingredients{
+  margin-left: 15%;
+  width: 70%;
+}
+.directions{
+  margin-left: 15%;
+  width: 70%;
+}
+
+/* --- component --- */
+
+.ingredients ul{
+  list-style-type: none;
+}
+.ingredients li{
+  list-style-type: none;
+}
+.ingredients ul > li {
+  text-indent: -20px;
+  color: grey;
+  font-size: 18px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: lighter;
+}
+.ingredients ul > li:before {
+  content: "-";
+  text-indent: -5px;
+}
+.directions ul{
+  list-style-type: decimal;
+  margin-top: 5%;
+}
+.directions ul > li{
```

Comment:
Just in case you are curious, there is a way to set your own counter with CSS, nothing to worry about, anyways.